uwuno/src/test/java/com/learning/uwuno/integration/tests/basicRoomTest.java

```diff
@@ -15,7 +15,7 @@
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class basicRoomTest {
-    private static ArrayList<String> runningRoomIds = new ArrayList<String>();
+    private final static ArrayList<String> runningRoomIds = new ArrayList<>();
```

Comment:
What is this array used for, I couldn't find anything with a search :(

Reply:
Nani? It's used in a few tests for clean-up (@AfterAll)

Reply:
Oops I was on wrong file. But why was this changed to final? We add new rooms to it in the tests.

Reply:
Oh I just read that final only prevents the reassigning of the variable not if it is changed, so I think you're right here and the other final arraylist. Sorry about that.

Reply:
Basically the variable can't be reassigned to something else. Intellij suggested it. It can still be modified